model/proto/model.proto

```diff
@@ -145,6 +145,9 @@ message Span {
     (gogoproto.customname) = "ProcessID"
   ];
   repeated string warnings = 12;
+  bool incomplete = 13 [
+    (gogoproto.nullable) = true
```

Comment:
I believe we can avoid dealing with bool pointers if we set nullable to `false`.  
Could you elaborate on why this needs to be nullable?

Reply:
Yeah, I think I misread this from stackoverflow: https://stackoverflow.com/questions/42622015/how-to-define-an-optional-field-in-protobuf-3/42634681

It is mentioned:
`In proto3, all fields are "optional" (in that it is not an error if the sender fails to set them)`

I can remove the nullable as it makes not much sense regarding this. But I think for the generated thrift it will still be a boolean pointer.